source/_integrations/fyta.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: FYTA
+description: Instructions on how to integrate FYTA sensors within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.1
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@dontinelli'
+ha_domain: fyta
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The **FYTA** {% term integration %} uses the open API of [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into Home Assistant.
+
+For the authentication on the FYTA server, you need your login-credentials (email and password).
+
+The integration provides a device for the controller interface and for all plants with a FYTA Beam sensor.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The following sensors are currently available for the controller:
+
+| name                  | Unit   | Description   |
+|-----------------------|--------|:-------------------------------------------|
+| plant_number          |        | Number of plants                           |
+
+The following sensors are currently available per plant:
+
+| name                  | Unit   | Description   |
+|-----------------------|--------|:-------------------------------------------|
+| last_update           |        | Time of latest data update.                |
+| plant_name            |        | Name of the plant (as set by user)         |
+| scientific_name       |        | Scientific name of the plant               |
+| plant_status          |        | FYTA-Status (number 1 to 5)                |
+| temperature_status    |        | FYTA-Status (number 1 to 5)                |
+| light_status          |        | FYTA-Status (number 1 to 5)                |
+| moisture_status       |        | FYTA-Status (number 1 to 5)                |
+| salinity_status       |        | FYTA-Status (number 1 to 5)                |
+| temperature           | °C     | Temperature measured by sensor             |
+| light                 | mol/d  | Light measured by sensor                   |
```

Comment:
is this referring to the day light integral DLI? 
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/812ab8a5-4923-449e-9dc5-5f45def913e1)

Reply:
No, the value provided by this API is the current value. It should also be possible to get the DLI, but this would be something for a new PR. While digging through the API and documentation, I noticed that I reported the wrong unit, just amended the documentation accordingly.

Reply:
I see. I was not trying to suggest to use DLI. I was simply trying to understand what I'm looking at. When I googled this, I always found it in connection with a certain area, for example per square meter (for example, in [this documentation on calculating light](https://edepot.wur.nl/156931)).

Reply:
I now also added an explanation of the unit